docs/integrations/pulse.md

```diff
@@ -0,0 +1,60 @@
+## Overview
+
+Pulse Connect Secure is an SSL VPN solution for remote and mobile users.
+
+## Setup
+
+This setup guide will show you how to forward logs produced by Pulse Connect Secure to SEKOIA.IO by means of an Rsyslog transport channel.
+
+On most GNU/Linux servers, two packages need to be installed: `rsyslog` and `rsyslog-gnutls`.
+
+### 1. Download the certificate
+
+In order to allow the connection of your rsyslog server to the SEKOIA.IO intake, please download the SEKOIA.IO intake certificate:
+
+```bash
+$ wget -O /etc/rsyslog.d/SEKOIA-IO-intake.pem https://app.sekoia.io/assets/files/SEKOIA-IO-intake.pem
+```
+
+### 2. Configure the rsyslog server
+
+Open or create a new Pulse Connect Secure configuration file for rsyslog:
+
+```bash
+sudo vim /etc/rsyslog.d/46-pulse.conf
+```
+
+Paste the following rsyslog configuration to trigger the emission of Pulse Connect Secure logs by your Rsyslog server to SEKOIA.IO:
+
+```bash
+# Define the SEKIOA-IO intake certificate
+$DefaultNetstreamDriverCAFile /etc/rsyslog.d/SEKOIA-IO-intake.pem
+
+# Configure up the network ssl connection
+$ActionSendStreamDriver gtls # use gtls netstream driver
+$ActionSendStreamDriverMode 1 # require TLS for the connection
+$ActionSendStreamDriverAuthMode x509/name # server is authenticated
+
+# Template definition [RFC5424](https://tools.ietf.org/html/rfc5424#section-7.2.2)
+# IMPORTANT: don't forget to set your intake key in the template
+template(name="SEKOIAIOPulseTemplate" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"YOUR_INTAKE_KEY\"] %msg%\n")
+
+# Send your events to SEKOIA.IO intake servers under SEKOIAIOPulseTemplate template
+if $programname startswith 'named' then @@(o)intake.sekoia.io:10514;SEKOIAIOPulseTemplate
```

Comment:
Is 'named' the real name, or is it a variable ?

Reply:
Oups, wrong copy/paste :)

Reply:
fixed